matchzoo/modules/stacked_brnn.py

```diff
@@ -0,0 +1,84 @@
+import torch
+import torch.nn as nn
+from torch.nn import functional as F
+
+
+class StackedBRNN(nn.Module):
+    """
+    Stacked Bi-directional RNNs.
+
+    Differs from standard PyTorch library in that it has the option to save
+    and concat the hidden states between layers. (i.e. the output hidden size
+    for each sequence input is num_layers * hidden_size).
+
+    Examples:
+        >>> import torch
+        >>> rnn = StackedBRNN(10, 10, 2, concat_layers=True)
```

Comment:
Set below will be better? Since it will cover all the code.
```suggestion
        >>> rnn = StackedBRNN(
        ...     input_size=10,
        ...     hidden_size=10,
        ...     num_layers=2,
        ...     dropout_rate=0.2,
        ...     dropout_output=True,
        ...     concat_layers=False
        ... )
```

Reply:
done